Test/WebTestCase.php

```diff
@@ -574,18 +572,7 @@ public function loadFixtureFiles(array $paths = array(), $append = false, $omNam
 
         $files = $this->locateResources($paths);
 
-        // Check if the Hautelook AliceBundle is registered and if yes, use it instead of Nelmio Alice
-        $hautelookLoaderServiceName = 'hautelook_alice.fixtures.loader';
```

Comment:
I can't recall if that is purged, if it is then the above service should be changed for `fidry_alice_data_fixtures.loader.doctrine`

Reply:
@theofidry You mean this? https://github.com/liip/LiipFunctionalTestBundle/pull/337/files#diff-f815866dba659cf4e7aebb6fc769c44cR558

Right?

Reply:
yep

Reply:
Well, there is nothing more to do in this case. :-)